bravado/response.py

```diff
@@ -93,3 +94,20 @@ def status_code(self):
 
     def json(self, **kwargs):
         return self._delegate.json(**kwargs)
+
+
+class FidoLibResponseAdapter(ResponseLike):
```

Comment:
s/FidoLib/Fido

The only request I used `RequestsLib` is because `Requests` is kind of very generic all by itself and not obvious that it referes to the "Request Library".